src/GeneralStateTestsFiller/stReturnDataTest/returndatacopy_initial_256Filler.json

```diff
@@ -0,0 +1,60 @@
+{
+    "returndatacopy_initial_256" : {
+        "env" : {
+            "currentCoinbase" : "0x2adc25665018aa1fe0e6bc666dac8fc2697ff9ba",
+            "currentDifficulty" : "0x20000",
+            "currentGasLimit" : "0x0a00000000",
+            "currentNumber" : "0x01",
+            "currentTimestamp" : "0x03e8",
+            "previousHash" : "0x5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6"
+        },
+        "expect" : [
+            {   "indexes" : {
+                   "data" : -1,
+                   "gas" : -1,
+                   "value" : -1
+                },
+                "network" : ["ALL"],
+                "result" : {
+                    "0x0f572e5295c57f15886f9b263e2f6d2d6c7b5ec6" : {
+                        "storage" : {
+                            "0x00" : "0x01"
+                        }
+                    }
+                }
+            }
+        ],
+        "pre" : {
+            "0x0f572e5295c57f15886f9b263e2f6d2d6c7b5ec6" : {
+                "balance" : "0x0de0b6b3a7640000",
+                "code" : "{ (MSTORE 0 0x112233445566778899aabbccddeeff) (RETURNDATACOPY (- 0 32) 0 32) (SSTORE 0 (MLOAD 0)) }",
```

Comment:
if you test the edge condition you could also check 
that test fails   on    X  case.   but it works on X-1 case. and fails on X+1 case. 
you could pass different value through transaction data